pkg/trace/agent/agent.go

```diff
@@ -30,6 +30,9 @@ import (
 )
 
 const (
+	// tagInferredSpanTagSourceKey is the key to the meta tag that lets us know whether this span should inherit its tags.
+	// Expected options are "lambda" and "self"
+	tagInferredSpanTagSourceKey = "_inferred_span.tag_source"
```

Comment:
```suggestion
	// tagInferredSpan is the key to the meta tag that lets us know whether this span should inherit its tags.
	// Expected options are "lambda" and "self"
	tagInferredSpan = "_inferred_span.tag_source"
```
Or, if you don't like my proposal, at least remove the `*Key` suffix.